apps/ledger-live-mobile/src/screens/CustomImage/Step0Welcome.tsx

```diff
@@ -29,33 +30,28 @@ const Step0Welcome: React.FC<
     setModalOpened(false);
   }, [setModalOpened]);
 
+  const { width } = useWindowDimensions();
+
   return (
     <SafeAreaView style={{ flex: 1 }} edges={["bottom"]}>
       <Flex flex={1}>
-        <Flex
-          backgroundColor="neutral.c40"
-          alignItems="center"
-          justifyContent="center"
-          height={250}
-          width="100%"
-        >
-          <Text>illustration placeholder</Text>
-        </Flex>
+        <Image
+          source={require("./assets/welcome.png")}
+          resizeMode="contain"
+          style={{ width, height: (256 / 375) * width }} // (image height / image width) * screen width
```

Comment:
is there a way of retrieving these dimensions automatically from the image? instead of hardcoding here maybe

Reply:
Yeah right, we do this in a bunch of places for images but maybe it's not a good practice at it would easily break if someone just replaces the asset. I'll change it